artman/cli/main.py

```diff
@@ -439,7 +439,6 @@ def _run_artman_in_docker(flags):
     root_dir = flags.root_dir
     output_dir = flags.output_dir
     artman_config_dirname = os.path.dirname(flags.config)
-    user_config = os.path.join(os.path.expanduser('~'), '.artman')
```

Comment:
I assume this is just because none of the user config is needed when running in docker?